modules/core/02-client/keeper/keeper.go

```diff
@@ -52,15 +58,29 @@ func (Keeper) Logger(ctx sdk.Context) log.Logger {
 	return ctx.Logger().With("module", "x/"+exported.ModuleName+"/"+types.SubModuleName)
 }
 
+// GetRouter returns the light client module router.
+func (k Keeper) GetRouter() *types.Router {
+	return k.router
+}
```

Comment:
As noted in thread https://github.com/cosmos/ibc-go/pull/5806/files#r1523681992

Refactor: `GetRouter()` can be changed to `ClientModule(clientID string)` or `Route(clientID string)` returning the light client module